x/evm/keeper/querier.go

```diff
@@ -183,31 +180,3 @@ func queryAccount(ctx sdk.Context, path []string, keeper Keeper) ([]byte, error)
 	}
 	return bz, nil
 }
-
-func queryExportAccount(ctx sdk.Context, path []string, keeper Keeper) ([]byte, error) {
```

Comment:
is there a reason for removing exportAccount functionality?

Reply:
because one should use the export command which exports the full genesis. Exporting a single account doesn't provide any use case